src/main/java/org/sonarlint/intellij/core/server/ServerLinks.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * SonarLint for IntelliJ IDEA
+ * Copyright (C) 2015-2023 SonarSource
+ * sonarlint@sonarsource.com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02
+ */
+package org.sonarlint.intellij.core.server
+
+import org.sonarlint.intellij.common.util.SonarLintUtils.withTrailingSlash
+import org.sonarlint.intellij.config.global.ServerConnection.SONARCLOUD_URL
+
+sealed interface ServerLinks {
```

Comment:
this could be replaced as a functional interface, not sure it make the later cleaner though :sweat_smile: